python_modules/dagster/dagster/_core/definitions/decorators/definitions_decorator.py

```diff
@@ -0,0 +1,19 @@
+from dagster._core.definitions.definitions_loader import DefinitionsLoader, DefinitionsLoadFn
+
+
+def definitions(fn: DefinitionsLoadFn) -> DefinitionsLoader:
```

Comment:
Additionally let's consider typing this as `Callable` with the appropriate param types rather than `DefinitionsLoader`, which I think we should treat as an internal framework detail.